bpf/kmesh/workload/include/config.h

```diff
@@ -9,7 +9,7 @@
 #define MAP_SIZE_OF_SERVICE  5000
 #define MAP_SIZE_OF_ENDPOINT 105000
 #define MAP_SIZE_OF_BACKEND  100000
-#define MAP_SIZE_OF_AUTH     8192
```

Comment:
I donot mean to reduce this size, it is used per socket. For xdp auth. 

What i meant is the per node auth policies can be small

Reply:
ok，I see what you mean. This size is also used in the auth-over-socket process. will change it